From 0c47f057ef35b7f72e5f3a3dea94e9734ea0c10d Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Fri, 26 Apr 2024 17:50:15 +0100
Subject: [PATCH 1069/1085] drm/panel-simple: Fix 7inch panel mode for
 misalignment

The 7inch panel is one line off the screen both horizontally
and vertically.

Alter the panel mode to correct this.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 drivers/gpu/drm/panel/panel-simple.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/gpu/drm/panel/panel-simple.c
+++ b/drivers/gpu/drm/panel/panel-simple.c
@@ -3421,11 +3421,11 @@ static const struct drm_display_mode ras
 	.hdisplay = 800,
 	.hsync_start = 800 + 59,
 	.hsync_end = 800 + 59 + 2,
-	.htotal = 800 + 59 + 2 + 46,
+	.htotal = 800 + 59 + 2 + 45,
 	.vdisplay = 480,
 	.vsync_start = 480 + 7,
 	.vsync_end = 480 + 7 + 2,
-	.vtotal = 480 + 7 + 2 + 21,
+	.vtotal = 480 + 7 + 2 + 22,
 	.flags = DRM_MODE_FLAG_NVSYNC | DRM_MODE_FLAG_NHSYNC,
 };
 
